app/i18n/en.json

```
@@ -79,7 +79,8 @@
     "showAllStations": "Show Full Route",
     "hideAllStations": "Hide Full Route",
     "noCalendarAccessTitle": "Calendar access disabled",
-    "noCalendarAccessMessage": "To add the ride to your calendar, open your device settings and grant access to your calendar."
+    "noCalendarAccessMessage": "To add the ride to your calendar, open your device settings and grant access to your calendar.",
+    "noPlatformSet": "No Platform set"
```

```suggestion
    "noPlatformSet": "Platform not assigned
```
Maybe that's clearer? If there's enough space in the layout.